ActionMenu/UIResources/index.js

```diff
@@ -570,6 +570,11 @@ function trigger_animation($el, animation) {
 }
 
 function selection_sector_set(sectors) {
+	// if there's only a single sector we should show the whole circle
+	if (sectors === 1) {
+		$sector.style.clipPath = `circle(50% at 50% 50%)`;
+		return;
+	}
 	const angle = pi2 / sectors;
 	const clip_path = compute_radial_mask(angle);
```

Comment:
Why not fix `compute_radial_mask` instead? It would benefit other usecases like radial. I think there was a kinda fix there too that could be removed (to check on my side).

Also if we want to show the full thing, what about clipPath btw?